bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    agent = ReActAgent.from_tools(query_engine_tools, llm=llm, verbose=True)
</pre><p>may explaining what react is and give a link to it, <a href="https://www.promptingguide.ai/techniques/react" target="_blank">https://www.promptingguide.ai/techniques/react</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Good point, added some text and a link to it</p>